core/sql/ustat/hs_globals.cpp

```diff
@@ -2887,6 +2890,22 @@ Lng32 HSGlobalsClass::Initialize()
         HSHandleError(retcode);
         TM->Commit(); // Must commit this transaction (even if schema didn't get created).
       }
+                                              /*=====================================*/
+                                              /*   CREATE HBASE STATS SCHEMA         */
+                                              /*   typically as trafodion.hbasestats */
+                                              /*=====================================*/
+    if (isNativeHbaseCat(objDef->getCatName()))
+      {
+        HSTranMan *TM = HSTranMan::Instance(); // Must have transaction around this.
+        TM->Begin("Create schema for native hbase stats.");
+        NAString ddl = "CREATE SCHEMA IF NOT EXISTS ";
+        ddl.append(HBASE_STATS_CATALOG).append('.').append(HBASE_STATS_SCHEMA);
```

Comment:
Do you want these schemas to be private or shared?  All tables in a private schema are owned by the creator.  Tables in a shared schema are owned by the creator. 
Who owns these schemas?  You should add a authorization clause to specify the owner.   You could make DB__ROOT the owner or add a new role as the owner. 
The MANAGE_STATISTICS is a privilege we support today.  Anyone with the MANAGE_STATISTICS privilege can perform update statistics.  Will this work today with sample tables.